### PR TITLE
refactor(css): variables and font sizes

### PR DESCRIPTION
**Pull request by gepifanio**
Related to the typography and colors override for FFAM.

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit b308447589fa885cf98a68d4a1be4a226536d088

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5cb034ed0c0d700008236306

File: components/_vars.css [hunk 1]
```diff
@@ -125,7 +125,6 @@
 
   /* FFAM VARIABLES */
```
Comment (TimMansell) [inline components/_vars.css]: Are these FFAM variables or can they be used in other websites?

Comment (gepifanio) [reply components/_vars.css]: This section was introduced at the beginning of FFAM project, when we were not sure if create a new repo for FFAM or just do it in `pattern-lib` so to build some components with the `FFAM` styling need that variables, could be a good idea to remove it, and move to `ffam-prototype`.

**Comment by gepifanio** [comment]
> Hi @gepifanio, what is the purpose of these changes? Just updating hard-coded colours to use the variables?

Yes, pretty much this.

**Pull request merged**